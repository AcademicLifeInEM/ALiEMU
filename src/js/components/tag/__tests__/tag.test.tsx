import React from 'react';
import renderer from 'react-test-renderer';

import Tag from '../index';

it('should render correctly', () => {
    const tree = renderer.create(<Tag text="Hello World" />);
    expect(tree).toMatchSnapshot();
});
