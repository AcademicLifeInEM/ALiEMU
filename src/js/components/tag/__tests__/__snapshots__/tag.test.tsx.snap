// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="tag"
  onClick={[Function]}
  onKeyDown={[Function]}
  role="button"
  tabIndex={-1}
>
  <span>
    Hello World
  </span>
</div>
`;
